src/utils/webrtc/webrtc.js

```diff
@@ -995,6 +956,42 @@ export default function initWebRTC(signaling, _callParticipantCollection, _local
 		sendDataChannelToAll('status', 'videoOff')
 	})
 
+	// Send the nick changed event via data channel and signaling
+	webrtc.on('nickChanged', function(name) {
+		let payload
+		if (signaling.settings.userId === null) {
```

Comment:
add comment to explain this

do we have different event arg formats ?

Reply:
Yes, `nickChanged` for guests only include the name as a string, while for users it is an object with the name as a string and the user ID as a string.

I thought that the code was self-explaining, but I can still add a comment if you want.